### PR TITLE
Add local up/create to README

### PR DESCRIPTION
**Pull request by SoManyHs**
Addresses #811 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1089,15 +1089,32 @@ Resource tags specified with `--tags` will be added to new IAM Roles and new or
...
+If you wish to run an ECS Task Definition specified in a different file, you can use the `--task-def-file` or `-f` flag with the name of the file.
+If you wish to run an ECS Task Definition already registered with ECS, you can use the `--task-def-remote` or `-t` flag with the ARN or family name of the Task Definition.
+You can also specify a different output file using the `--output` or `-o` flag.
+
 
 #### ecs-cli local up
+If you want to convert an ECS Task Definition to a Docker Compose file, you can run:
+
+```
+$ ecs-cli local create
```
Comment (iamhopaul123) [inline README.md]: 1. Should this be `$ ecs-cli local up`?
2. Should we put more info about `up`, since `up` not only "convert" but also "run" containers?

Comment (efekarakus) [reply README.md]: Agreed, I think we can merge this PR for "local create" and create a separate one for "local up". What do you think @SoManyHs ?

For local up, it'd be nice to tell users about:
- The [Amazon ECS Local Container Endpoints](https://github.com/awslabs/amazon-ecs-local-container-endpoints) and the `ecs-local-network`.
- Overrides as a way of replacing a container that's in a task definition with a local one.

Comment (SoManyHs) [reply README.md]: > Should this be $ ecs-cli local up?

I mean, this does all apply for create. I do think that we can flesh out local up more, but if the implication here is that the whole description should only apply to up I'm not sure I agree.

Comment (SoManyHs) [reply README.md]: Oh. I just realized my PR is not what I have latest locally. Plz hold

**Pull request merged**